src/racing/actors.c

```diff
@@ -2816,88 +1725,88 @@ s32 func_8029F69C(Player *player, struct Actor *actor) {
         player->pos[0] += x_dist * temp_f2 * 0.5f;
         player->pos[2] += z_dist * temp_f2 * 0.5f;
     }
-    return 1;
+    return TRUE;
 }
 
-s32 func_8029FB80(Player *arg0, struct Actor *arg1) {
+bool is_collision_between_item_player(Player *arg0, struct Actor *arg1) {
     f32 temp_f0;
-    f32 temp_f12_2;
-    f32 temp_f14;
-    f32 temp_f16;
-    f32 temp_f2;
+    f32 dist;
+    f32 yDist;
+    f32 zDist;
+    f32 xDist;
 
     temp_f0 = arg0->boundingBoxSize + arg1->boundingBoxSize;
-    temp_f2 = arg1->pos[0] - arg0->pos[0];
-    if (temp_f0 < temp_f2) {
-        return 0;
+    xDist = arg1->pos[0] - arg0->pos[0];
+    if (temp_f0 < xDist) {
+        return FALSE;
     }
-    if (temp_f2 < -temp_f0) {
-        return 0;
+    if (xDist < -temp_f0) {
+        return FALSE;
     }
-    temp_f14 = arg1->pos[1] - arg0->pos[1];
-    if (temp_f0 < temp_f14) {
-        return 0;
+    yDist = arg1->pos[1] - arg0->pos[1];
+    if (temp_f0 < yDist) {
+        return FALSE;
     }
-    if (temp_f14 < -temp_f0) {
-        return 0;
+    if (yDist < -temp_f0) {
+        return FALSE;
     }
-    temp_f16 = arg1->pos[2] - arg0->pos[2];
-    if (temp_f0 < temp_f16) {
-        return 0;
+    zDist = arg1->pos[2] - arg0->pos[2];
+    if (temp_f0 < zDist) {
+        return FALSE;
     }
-    if (temp_f16 < -temp_f0) {
-        return 0;
+    if (zDist < -temp_f0) {
+        return FALSE;
     }
-    temp_f12_2 = (temp_f2 * temp_f2) + (temp_f14 * temp_f14) + (temp_f16 * temp_f16);
-    if (temp_f12_2 < 0.1f) {
-        return 0;
+    dist = (xDist * xDist) + (yDist * yDist) + (zDist * zDist);
+    if (dist < 0.1f) {
+        return FALSE;
     }
-    if ((temp_f0 * temp_f0) < temp_f12_2) {
-        return 0;
+    if ((temp_f0 * temp_f0) < dist) {
+        return FALSE;
     }
-    return 1;
+    return TRUE;
 }
 
-s32 func_8029FCA4(struct Actor *arg0, struct Actor *arg1) {
+bool is_two_actor_collide(struct Actor *arg0, struct Actor *arg1) {
```

Comment:
query_collision_actor_vs_actor_aabb

Reply:
rename it query_collision_actor_vs_actor because I don't know what aabb mean